src/Fetchers/TMSFetcher/TMSMatchFetcher.ts

```diff
@@ -46,6 +46,23 @@ export class TMSMatchFetcher {
             matches.set(item.getID(), item);
         }
 
+        // Fetch officials data
```

Comment:
Instead of implementing this in the `TMSMatchFetcher.ts` class, please create a separate `TMSOfficialFetcher.ts` class and move all official-related logic into there.